.travis.yml

```diff
@@ -20,3 +20,4 @@ env:
   - TEST_SUITE=test:karma
   - TEST_SUITE=test:testcafe-ci
   - TEST_SUITE=test:jest
+  - TEST_SUITE=test:enleaks-ci
```

Comment:
I'd argue we don't need to run this in Travis, as external developers _usually_ are not creating new i18n resource strings and/or Widget views. Obviously a larger topic - but for now can we scope this to just our internal CI tool?

Reply:
Good callout. I am ok not run this on travis.